fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -236,32 +238,56 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         }
     }
 
-    private fun initTranslationsAction(context: Context) {
+    private fun initTranslationsAction(context: Context, view: View) {
         if (!context.settings().enableTranslations) {
             return
         }
 
-        val translationsAction =
-            BrowserToolbar.ToggleButton(
-                image = AppCompatResources.getDrawable(
-                    context,
-                    R.drawable.mozac_ic_translate_24,
-                )!!.apply {
-                    setTint(ContextCompat.getColor(context, R.color.fx_mobile_text_color_primary))
-                },
-                imageSelected = AppCompatResources.getDrawable(
-                    context,
-                    R.drawable.mozac_ic_translate_24,
-                )!!,
-                contentDescription = context.getString(R.string.browser_toolbar_translate),
-                contentDescriptionSelected = "",
-                visible = {
-                    translationsAvailable || context.settings().enableTranslations
-                },
-                listener = { browserToolbarInteractor.onTranslationsButtonClicked() },
-            )
-
+        val translationsAction = Toolbar.ActionButton(
+            AppCompatResources.getDrawable(
+                context,
+                R.drawable.mozac_ic_translate_24,
+            ),
+            contentDescription = context.getString(R.string.browser_toolbar_translate),
```

Comment:
This sounded right in TalkBack and the color change both tested well for me when checking states!